site/gatsby-site/config.js

```diff
@@ -210,4 +212,23 @@ const config = {
   },
 };
 
+const requiredEnvs = {
+  GATSBY_SITE_URL: process.env.GATSBY_SITE_URL,
+  GATSBY_AVAILABLE_LANGUAGES: process.env.GATSBY_AVAILABLE_LANGUAGES,
+  // GATSBY_ALGOLIA_SEARCH_KEY: process.env.GATSBY_ALGOLIA_SEARCH_KEY,
+  // GATSBY_ALGOLIA_APP_ID: process.env.GATSBY_ALGOLIA_APP_ID,
+  // TODO we should validate these too but I don't want to do that in this PR
+  // ALGOLIA_ADMIN_KEY: process.env.ALGOLIA_ADMIN_KEY,
+  // GOOGLE_MAPS_API_KEY: process.env.GOOGLE_MAPS_API_KEY,
+  // GOOGLE_TRANSLATE_API_KEY: process.env.GOOGLE_TRANSLATE_API_KEY,
+  // GATSBY_ROLLBAR_TOKEN: process.env.GATSBY_ROLLBAR_TOKEN,
+  // CLOUDFLARE_R2_ACCOUNT_ID: process.env.CLOUDFLARE_R2_ACCOUNT_ID,
+  // CLOUDFLARE_R2_ACCESS_KEY_ID: process.env.CLOUDFLARE_R2_ACCESS_KEY_ID,
+  // CLOUDFLARE_R2_SECRET_ACCESS_KEY: process.env.CLOUDFLARE_R2_SECRET_ACCESS_KEY,
+  // CLOUDFLARE_R2_BUCKET_NAME: process.env.CLOUDFLARE_R2_BUCKET_NAME,
+  // GATSBY_CLOUDFLARE_R2_PUBLIC_BUCKET_URL: process.env.GATSBY_CLOUDFLARE_R2_PUBLIC_BUCKET_URL,
```

Comment:
We should remove these commented lines if they are not required env vars

Reply:
They are, but uncommenting them requires more changes here and there, and I don't want to make the PR larger.